posts/2023-06-20-introducing-leadership-council.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: "Introducing the Rust Leadership Council"
+author: Leadership Council
+team: Leadership Council <https://www.rust-lang.org/governance/teams/leadership-council>
+---
+
+As of today, [RFC 3392] has been merged, forming the new top level governance body of the Rust Project: the Leadership Council. The creation of this Council marks the end of both the Core Team and the interim Leadership Chat.
+
+The Council will assume responsibility for top-level governance concerns while most of the responsibilities of the Rust Project (such as maintenance of the compiler and core tooling, evolution of the language and standard libraries, administration of infrastructure, etc.) remain with the nine top level teams.
+
+Each of these top level teams has chosen a representative who collectively form [the Council](https://www.rust-lang.org/governance/teams/leadership-council):
+- Compiler: Eric Holk
+- Crates.io: Carol (Nichols || Goulding)
+- Dev Tools: Eric Huss
+- Infrastructure: Ryan Levick
+- Language: Jack Huey
+- Launching Pad: Jonathan Pallant
```

Comment:
a lot of people were confused about launching pad earlier: might be worth including a small footnote explaining what it is and linking to the RFC?

Reply:
+1

Reply:
How necessary do you think this is?

The RFC is linked above, so its easily available.

I could imagine adding an extra link "as defined by the RFC" after the "Each of the top level teams" above, but we will have to restart the approval vote.

Reply:
I think I've seen Launching Pad come up quite often in discussions of public comms around leadership stuff and people are always confused because suddenly there's a new team they've never heard of, and people don't expect it to be _new_ they just think they missed it's announcement and are confused.

People will not necessarily expect it to be in the RFC. Even a small pointer would probably help.


(in the long run y'all should probably figure out a way to draft comms that allows for minor editorial changes to not require full approval. I have ideas, but I'll get to that once y'all are properly up and running)

Reply:
I don't think it's _crucial_, if there's a rush we should just get this out, but a general problem to look out for in such comms to the general community is assuming too much since most community members do not pay attention to the minor details.